ansible/roles/containerd/tasks/install.yaml

```
@@ -15,7 +15,7 @@
   when:
     - not containerd_tar_file_exists.stat.exists
 
-- name: download containerd
+- name: download containerd for {{ containerd_tar_file }}
```

nit: is this name better?
```suggestion
- name: download containerd {{ containerd_tar_file }}
```

Also do we still want this, will it be displayed correctly when air-gapped with `containerd_tar_file_exists.stat.exists=true`?

---

i think the "for" in this case adds a bit more context so i prefer to keep it